cmd/namespace.go

```diff
@@ -15,7 +15,7 @@ func newNamespaceCommand() *cobra.Command {
 	}
 
 	createCmd := &cobra.Command{
-		Use:   "create [name] [vcs] [org-name]",
+		Use:   "create NAME github|bitbucket ORG-NAME",
 		Short: "create an namespace",
```

Comment:
This needs more docs about what an org name is, and how to know whether to specify GitHub/ButBucket.

Reply:
There is a card to add positional args to the `help` docs for command usage, that should clear this up. I'm afraid of adding too much to the `use` line that would just become clutter.